documentation/develop/temporary-installation-in-firefox.md

```diff
@@ -76,26 +79,27 @@ The extension will be installed, and will stay installed until you restart Firef
 
 ## Reloading a temporary extension
 
-Starting in Firefox 48, there's a new button labeled "Reload" next to the extension's entry in about:debugging:
+To reload the extension, click "Reload":
 
 ![reload-extension]({% asset "documentation/develop/reload-extension.png" @optim @path %})
 
-This does what it says:
+This:
 
-- reloading any persistent scripts, such as [background scripts](https://developer.mozilla.org/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#Background_scripts)
-- parsing the `manifest.json` file again, so changes to [`permissions`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions), [`content_scripts`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts), [`browser_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action) or any other keys will take effect.
+- reloads any persistent scripts, such as [background scripts](https://developer.mozilla.org/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#Background_scripts)
+- parses the `manifest.json` file, so changes to [`permissions`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions), [`content_scripts`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts), [`browser_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action), or any other keys take effect.
 
 <!-- Video Popup Thumbnail -->
 
 {% include modules/video-popup.html
     title="borderify WebExtension - part 3"
-    youtube_id="NuajE60jfGY"
+    youtube_id="d3hgLMVJAzY"
     image="documentation/develop/borderify-part-3.png"
     alt="borderify part 3"
 %}
 
 <!-- END: Video Popup Thumbnail -->
 
+<!-- 
```

Comment:
Does this need to be commented or can it be removed?

Reply:
This can be removed.